netlify.toml

```
@@ -1,5 +1,5 @@
 [build]
-  command = "yarn rw db up --no-db-client --auto-approve && yarn rw dataMigrate up && yarn rw build"
+  command = "yarn rw db up --no-db-client --auto-approve && yarn rw build"
```

```suggestion
  command = "yarn rw db up --no-db-client --auto-approve && yarn rw build && yarn rw dataMigrate up"
```
Could something like this work? Since at this point the db client should be built.